code/game/objects/items/cards_ids.dm

```diff
@@ -96,6 +96,28 @@
 		return
 	A.emag_act(user)
 
+//emaging an emag
+/obj/item/weapon/card/emag/emag_act(mob/user)
+	var/otherEmag
+	if(user.l_hand)
+		if(istype(user.l_hand, /obj/item/weapon/card/emag))
+			if(user.l_hand != src)
+				otherEmag = user.l_hand
+	if(!otherEmag)
+		if(user.r_hand)
+			if(istype(user.r_hand, /obj/item/weapon/card/emag))
+				if(user.r_hand != src)
+					otherEmag = user.r_hand
+	if(!otherEmag)
+		return
+	to_chat(user, "<span class='notice'>The cyptographic sequencers attempt to override each other inevitably destroying themselves.</span>")
+	playsound(src.loc, "sparks", 50, 1)
+	new /mob/living/simple_animal/pet/dog/corgi(src.loc) //a doggo appear as a consultation prize :3

```

Comment:
????

Reply:
you confused at the corgi or did I do something horrifically wrong